配置/默认/HGLR-HGLRCF722.config

```diff
@@ -50,9 +50,7 @@ resource PINIO 2 C09
 resource FLASH_CS 1 D02
```

Comment:
You are moving this file into a different directory - this will not work, and the target will become unavailable to users.